src/components/ListOfCards/ListOfCards.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import ProductCard from "../ProductCard/ProductCard"
+import SellerCard from "../SellerCard/SellerCard";
+import styles from './ListOfCards.module.scss';
+
+export default function ListOfCards({ items, title }) {
```

Comment:
i'd suggest to thing of a better name for this component, to be more descriptive, for example: containers/ProcuctCards

Reply:
...but ListOfCards is a component that displays list of <ProductCard> or list of <SellerCard> depending what is passed in props

Reply:
done